docs/sources/flow/reference/components/otelcol.processor.attributes.md

```diff
@@ -164,6 +164,10 @@ For example, adding a `span_names` filter could cause the component to error if
 
 The `exclude` block provides an option to exclude data from being fed into the [action] blocks based on the properties of a span, log, or metric records.
```

Comment:
Is this true?
```suggestion
The `exclude` block provides an option to exclude data from being fed into the [action] blocks based on the properties of a span, log, or metric records. Excluded data will be forwarded as-is.
```

Reply:
It is true, but I will add it to the new block that I'm adding now, since clarifying this is the whole purpose of the block. I'm also not sure if readers would know what is meant by "forwarded". I'll change this:

> Signals excluded by the `exclude` block will still be propagated to downstream components.

To:

> Signals excluded by the `exclude` block will still be propagated to downstream components as-is.